matomo/content.md

```diff
@@ -50,25 +50,24 @@ The following environment variables are also honored for configuring your Matomo
 -	`MATOMO_DATABASE_PASSWORD`
 -	`MATOMO_DATABASE_DBNAME`
 
+The PHP memory limit can be configured with the following environment variable:
+
+-	`PHP_MEMORY_LIMIT`
+
 ## Docker-compose examples and log import instructions
 
-A minimal set-up using docker-compose is available in the [.examples folder](https://github.com/matomo-org/docker/tree/master/.examples).
+A minimal set-up using docker-compose is available in the [.examples folder](https://github.com/matomo-org/docker/tree/master/.examples), a more complete [example can be found at IndieHosters/matomo](https://github.com/libresh/compose-matomo/blob/master/docker-compose.yml).
```

Comment:
This new example seems to be 6 years old -- is it definitely still accurate and more complete than what's in the matomo org?